airbyte-integrations/bases/base-normalization/normalization/transform_catalog/destination_name_transformer.py

```diff
@@ -29,6 +29,9 @@
     DestinationType.CLICKHOUSE.value: 63,
     # https://docs.pingcap.com/tidb/stable/tidb-limitations
     DestinationType.TIDB.value: 64,
+    # According to the DuckDB team there no restriction: We don't enforce a maximum right now but I would not recommend having column names
+    # longer than a few kilobytes. https://discord.com/channels/909674491309850675/1067042662827438122/1067043835768737893.
+    DestinationType.DUCKDB.value: 255,
```

Comment:
@edgao Although there is no limitation from the DuckDB side. As soon as I change it to e.g. 255, three unit tests fail:
```
         =========================== short test summary info ============================
         FAILED unit_tests/test_table_name_registry.py::test_resolve_names[DestinationType.DUCKDB-long_name_truncate_collisions_catalog]
         FAILED unit_tests/test_table_name_registry.py::test_resolve_names[DestinationType.DUCKDB-un-nesting_collisions_catalog]
         FAILED unit_tests/test_table_name_registry.py::test_resolve_names[DestinationType.DUCKDB-nested_catalog]
```

Do you know the correlation? I changed it back to `64` and it works. So I will leave with this except you would know an explanation.


More detailed error:
```
        E         ?                                                                                                                     +++ ++++ ++++++++
         E         -  'schema_test.adcreatives_object_story_spec_link_data_image_crops_400x150.400x150': {'file': 'adcreatives_object_st__a_image_crops_400x150',
         E         +  'schema_test.adcreatives_object_story_spec_link_data_image_crops_400x150.400x150': {'file': 'adcreatives_object_story_spec_link_data_image_crops_400x150',
         E         ?                                                                                                                    +++ ++++ ++++++++
         E                                                                                                'schema': 'schema_test',
         E         -                                                                                      'table': 'adcreatives_object_st__a_image_crops_400x150'},
         E         +                                                                                      'table': 'adcreatives_object_story_spec_link_data_image_crops_400x150'},
         E         ?                                                                                                                     +++ ++++ ++++++++
         E         -  'schema_test.adcreatives_object_story_spec_link_data_image_crops_400x500.400x500': {'file': 'adcreatives_object_st__a_image_crops_400x500',
         E         +  'schema_test.adcreatives_object_story_spec_link_data_image_crops_400x500.400x500': {'file': 'adcreatives_object_story_spec_link_data_image_crops_400x500',
         E         ?                                                                                                                    +++ ++++ ++++++++
         E                                                                                                'schema': 'schema_test',
         E         -                                                                                      'table': 'adcreatives_object_st__a_image_crops_400x500'},
         E         +                                                                                      'table': 'adcreatives_object_story_spec_link_data_image_crops_400x500'},
         E         ?                                                                                                                     +++ ++++ ++++++++
         E         -  'schema_test.adcreatives_object_story_spec_link_data_image_crops_600x360.600x360': {'file': 'adcreatives_object_st__a_image_crops_600x360',
         E         +  'schema_test.adcreatives_object_story_spec_link_data_image_crops_600x360.600x360': {'file': 'adcreatives_object_story_spec_link_data_image_crops_600x360',
         E         ?                                                                                                                    +++ ++++ ++++++++
         E                                                                                                'schema': 'schema_test',
         E         -                                                                                      'table': 'adcreatives_object_st__a_image_crops_600x360'},
         E         +                                                                                      'table': 'adcreatives_object_story_spec_link_data_image_crops_600x360'},
         E         ?                                                                                                                     +++ ++++ ++++++++
         E         -  'schema_test.adcreatives_object_story_spec_link_data_image_crops_90x160.90x160': {'file': 'adcreatives_object_st__ta_image_crops_90x160',
         E         +  'schema_test.adcreatives_object_story_spec_link_data_image_crops_90x160.90x160': {'file': 'adcreatives_object_story_spec_link_data_image_crops_90x160',
         E         ?                                                                                                                  +++ ++++ +++++++
         E                                                                                              'schema': 'schema_test',
         E         -                                                                                    'table': 'adcreatives_object_st__ta_image_crops_90x160'},
         E         +                                                                                    'table': 'adcreatives_object_story_spec_link_data_image_crops_90x160'},
         E         ?                                                                                                                   +++ ++++ +++++++
         E           }

```

Reply:
Just checking but should this value be `64` since you've mentioned above that it doesn't work when the value is set to `255`?